presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -88,6 +88,11 @@ String Functions
     If ``search`` is an empty string, inserts ``replace`` in front of every
     character and at the end of the ``string``.
 
+.. function:: replace_first(string, search, replace) -> varchar
+    Replaces first instances of ``search`` with ``replace`` in ``string``.
+
+    If ``search`` is an empty string, inserts ``replace`` at the beginning of the ``string``.
```

Comment:
nit: If ``search`` is an empty string, it inserts ``replace`` at the beginning of the ``string``.